src/main/java/com/example/demo/feed/service/CommentService.java

```diff
@@ -0,0 +1,52 @@
+package com.example.demo.feed.service;
+
+
+import com.example.demo.feed.domain.Comment;
+import com.example.demo.feed.domain.Feed;
+import com.example.demo.feed.domain.FeedRepository;
+import com.example.demo.feed.domain.Writer;
+import com.example.demo.user.domain.User;
+import com.example.demo.user.domain.UserRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.Optional;
+
+
+@Service
+@RequiredArgsConstructor
+public class CommentService {
+
+    private final FeedRepository feedRepository;
+
+    private final UserRepository userRepository;
+
+    @Transactional
+    public void makeCommentByFeedIdAndContents(Long feedId, String contents) {
+        Authentication authentication = SecurityContextHolder.getContext().getAuthentication();
+        Feed feed = feedRepository.findFeedByIdAndIsDeleted(feedId, false);
+        User user = userRepository.findByUserId(authentication.getName());
+        Writer writer = Writer.makeWriter(user);
+
+        Comment comment = Comment.makeComment(writer, contents);
+        feed.enrollComment(comment);
+    }
+
+    @Transactional
+    public String likeOrDislikeCommentByFeedIdAndCommentId(Long feedId, int commentId) {
+        User user = null;
+        try {
+            Feed feed = feedRepository.findFeedByIdAndIsDeleted(feedId, false);
+            Comment comment = feed.getComments().get(commentId);
+            if(comment.checkPossibleOfLike(user)){
+                comment.likeOrDislike();
+            }
+        } catch (Exception e){
+            return "400 Bad Request";
+        }
+        return null;
```

Comment:
메소드에 직접 null을 return을 시키지말고 반환형을 `Optional<String>`로 바꿔서 보내는 건 어떨까요?
```suggestion
        return Optional.empty();
```